### ISSUE TITLE
Regex in schema json prevent connaisseur to validate signers with dash

### ISSUE THREAD

**Issue by Athosone** [sse-secure-systems/connaisseur#74]
**Describe the bug**

We have a container registry in azure where the signer's name is "mysigner-dev".

When we try to deploy an application from this registry the **snapshot.json** returns the list of the targets it includes the **target/mysigner-dev** the image is rejected with this error message:

```
ERROR: {'message': 'trust data has invalid format.', 'context': {'trust_data_type': 'snapshot', 'user': 'system:serviceaccount:ns:sa', 'operation': 'CREATE', 'kind': 'Deployment', 'namespace':
'mywebapp', 'name': 'mywebapp', 'image': 'myregistry.azurecr.io/webapp:0.1.20.0'}, 'detection_mode': True}
```

After some research, it appears that the code uses a regex which does not handle anything but words.

**Expected behavior**

Images are not rejected.

**Optional: To reproduce**

Add a signer with a "-" in its name or any char which is not a word as per the regex provided in the snapshot schema:

```
                "meta": {
                    "type": "object",
                    "patternProperties": {
                        "^(root|targets(\/\\w*)?)$": {
```

This can be easily reproduced through your unit tests.

**Optional: Versions (please complete the following information as relevant):**
 - Kubernetes Cluster: k3s 1.19.4
 - Notary Server: azure
 - Container registry: azure
 - Connaisseur: latest (1.4.0)

------

Ps: Your tool is really great and well made, thanks for the work you put into it! 💯

**Comment by phbelitz** [comment]
Thank you very much for the issue. We are planning to fix this on the upcomming release, which we'll hopefully release by tomorrow morning!

**Comment by Athosone** [comment]
Wow that was fast, thanks to you for the quick fix :)!

Ok great I ll update once its available

**Comment by xopham** [comment]
Hey @Athosone, it's available now! Part of the newest release [v1.4.4](https://github.com/sse-secure-systems/connaisseur/releases/tag/v1.4.4). We hope it works for you. Thanks for the feedback!

**Comment by Athosone** [comment]
Tested and integrated it yesterday!
Works perfectly thanks for the fix <3
